packages/breakdown-trees/package.json

```diff
@@ -0,0 +1,141 @@
+{
+  "name": "@bentley/breakdown-trees-react",
+  "version": "0.1.0",
+  "description": "Package for breakdown trees",
+  "keywords": [
+    "Bentley",
+    "iModel breakdown Tree",
+    "iModel.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/breakdown-trees"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/breakdown-trees-react.js",
+  "typings": "lib/breakdown-trees-react",
+  "scripts": {
+    "build": "npm run copy:assets && tsc --sourceMap 1>&2",
+    "build-prod": "tsc 1>&2 && npm run copy:assets",
+    "rebuild": "npm run clean && npm run build",
+    "makegraph": "depcruise --exclude \"(node_modules|src/test)\" --output-type dot src > graphs/dependencygraph.svg",
+    "copy:assets": "cpx \"./src/assets/**/*\" ./lib/assets && cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "build:watch": "tsc -w",
+    "clean": "rimraf lib",
+    "cover": "",
+    "test": "",
+    "test:watch": "npm test -- --reporter min --watch-extensions ts,tsx --watch",
+    "lint": "tslint --project . 1>&2",
+    "lintfix": "tslint --fix --project . 1>&2"
+  },
+  "dependencies": {
+    "classnames": "^2.3.1",
+    "i18next": "^10.2.2",
+    "react-redux": "^7.2.0",
+    "redux": "^4.0.3"
+  },
+  "devDependencies": {
+    "@bentley/bentleyjs-core": "^2.15.0",
+    "@bentley/build-tools": "~2.16.0",
+    "@bentley/geometry-core": "^2.15.0",
+    "@bentley/imodeljs-common": "^2.15.0",
+    "@bentley/imodeljs-frontend": "^2.15.0",
+    "@bentley/imodeljs-i18n": "^2.15.0",
+    "@bentley/imodeljs-markup": "^2.15.0",
+    "@bentley/imodeljs-quantity": "^2.15.0",
+    "@bentley/itwin-client": "^2.15.0",
+    "@bentley/presentation-common": "^2.15.0",
+    "@bentley/presentation-components": "^2.15.0",
+    "@bentley/presentation-frontend": "^2.15.0",
+    "@bentley/presentation-testing": "^2.15.0",
+    "@bentley/presentation-backend": "^2.15.0",
+    "@bentley/product-settings-client": "^2.15.0",
+    "@bentley/ui-abstract": "^2.15.0",
+    "@bentley/ui-components": "^2.15.0",
+    "@bentley/ui-core": "^2.15.0",
+    "@bentley/ui-framework": "^2.15.0",
+    "@bentley/ui-ninezone": "^2.15.0",
+    "@bentley/webgl-compatibility": "^2.15.0",
+    "@bentley/webpack-tools": "1.14.1",
+    "@bentley/measure-tools-react": "workspace:*",
```

Comment:
idk if we want to be explicitly depending on other pkgs in the repo like this. might be ok, but probably want to use ^ for the peer dep